### DISCUSSION TITLE
HTTPS support

### DISCUSSION THREAD

**Discussion by SofieneOuerdiane** [apollographql/router#2724] [Q&A] [answered]
Hello,

When I instantiate a container of Apollo Router from the image `ghcr.io/apollographql/router`. The default exposed port is `4000`. And the default protocol  is HTTP. Does apollographql router support `HTTPS` ?
 Can it listen for two ports (one for `HTTP` and another one for `HTTPS`) ? I searched for answers in the official docs but I could not find any answers ? Does anyone have any idea ? Thanks !

**Comment by bnjjj** [comment, marked as answer]
It has been implemented few days ago, cf https://github.com/apollographql/router/issues/2615 and will be part of the next release
